.github/workflows/prerelease.yml

```diff
@@ -88,127 +32,12 @@ jobs:
           GOPATH: ${{ github.workspace }}
         run: make integration-test
 
-  prerelease:
-    name: Build binary for *Nix/Win, create archives for *Nix/Win, create packages for *Nix, upload all artifacts into GH Release assets
-    runs-on: ubuntu-20.04
-    needs: [test-nix, test-windows, snyk, test-integration-nix]
-    env:
-      GPG_MAIL: 'infrastructure-eng@newrelic.com'
-      GPG_PASSPHRASE: ${{ secrets.OHAI_GPG_PASSPHRASE }}
-      GPG_PRIVATE_KEY_BASE64: ${{ secrets.OHAI_GPG_PRIVATE_KEY_BASE64 }} # base64 encoded
-    steps:
-      - uses: actions/checkout@v2
-      - name: Login to DockerHub
-        uses: docker/login-action@v3
-        with:
-          username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
-          password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
-      - name: Pre release
-        run: make ci/prerelease
-      - name: Notify failure via Slack
-        if: ${{ failure() }}
-        uses: archive/github-actions-slack@master
-        with:
-          slack-bot-user-oauth-access-token: ${{ secrets.COREINT_SLACK_TOKEN }}
-          slack-channel: ${{ secrets.COREINT_SLACK_CHANNEL }}
-          slack-text: "❌ `${{ env.REPO_FULL_NAME }}`: prerelease pipeline failed."
-
-  package-win:
-    name: Create MSI & Upload into GH Release assets
-    runs-on: windows-2019
-    needs: [prerelease]
-    env:
-      GOPATH: ${{ github.workspace }}
-      PFX_CERTIFICATE_BASE64: ${{ secrets.OHAI_PFX_CERTIFICATE_BASE64 }} # base64 encoded
-      PFX_CERTIFICATE_DESCRIPTION: 'New Relic'
-      PFX_PASSPHRASE:  ${{ secrets.OHAI_PFX_PASSPHRASE }}
-    defaults:
-      run:
-        working-directory: src/github.com/${{env.ORIGINAL_REPO_NAME}}
-    strategy:
-      matrix:
-        goarch: [ amd64 ] # 386 not supported in jmx integrations
-        test-upgrade: [true,false]
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          path: src/github.com/${{env.ORIGINAL_REPO_NAME}}
-      - name: Get PFX certificate from GH secrets
-        shell: bash
-        run: printf "%s" "$PFX_CERTIFICATE_BASE64" | base64 -d - > wincert.pfx
-      - name: Download zip from GH Release assets and extract .exe
-        shell: pwsh
-        run: |
-          build\windows\download_zip_extract_exe.ps1 "$env:INTEGRATION" ${{ matrix.goarch }} "$env:TAG" "$env:REPO_FULL_NAME"
-      - name: Download nrjmx
```

Comment:
This step seems pretty specific, is it triggered when using the reusable-workflow?

Reply:
It seems to be triggered due to `windows_download_nrjmx: true` usage